fv/workload/workload.go

```diff
@@ -121,6 +121,16 @@ func Run(c *containers.Container, name, interfaceName, ip, ports string, protoco
 			log.Infof("Workload %s stderr: %s", name, strings.TrimSpace(string(line)))
 		}
 	}()
+
+	namespacePath, err := stdoutReader.ReadString('\n')
+	if err != nil {
+		// (Only) if we fail here, wait for the stderr to be output before returning.
+		defer errDone.Wait()
```

Comment:
Just wondering what difference it makes to use `defer` here, instead of directly calling `errDone.Wait()` ?

Reply:
Think the idea was that failing the test first makes it easier to diagnose if the Wait() call hangs.  Pretty spurious though!